frontend/src/components/roadmap/RoadmapPageSection.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+interface RoadmapPageSectionProps {
+  sectionContent: React.ReactNode;
+  sectionTitle?: string;
+  extraClasses?: string;
+}
+
+export default function RoadmapPageSection({
+  sectionContent,
+  sectionTitle = "",
+  extraClasses = "",
+}: RoadmapPageSectionProps) {
```

Comment:
A couple of recommendations here. To clean it up a little you could remove the `extraClasses` as a prop and just add `className` to  the implementations: `<RoadmapPageSection className="bg-white"/>`.  `sectionTitle` could be simplified as `title` as it conveys the same meaning with less. Finally the `sectionContent` could be changed to `{children}` which would allow wrapping whatever the implementation wants to pass as JSX.